test/Constraints/rdar82828226.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: %target-typecheck-verify-swift
+
+import Foundation
```

Comment:
In order to import Foundation we would need objc_interop
```suggestion
// REQUIRES: objc_interop

import Foundation
```